### PR TITLE
Add netcore50 configuration to System.Net.Primitives

### PR DESCRIPTION
**Pull request by mellinoe**
Also consolidate the net46 project in the 'facade' directory down into the root project file.

@davidsh , @CIPop , @weshaggard

### CODE REVIEW

File: src/System.Net.Primitives/src/project.json [hunk 1]
```diff
@@ -1,21 +1,47 @@
...
+        "System.Runtime.InteropServices": "4.0.20",
+        "System.Runtime.WindowsRuntime": "4.0.0",
+        "System.Threading": "4.0.0",
+        "System.Threading.Tasks": "4.0.0",
+        "Microsoft.TargetingPack.Private.WinRT": "1.0.0",
+        "Microsoft.Win32.Primitives": "4.0.0",
+      }
+    }
   }
 }
```
Comment (davidsh) [inline src/System.Net.Primitives/src/project.json]: missing newline at end of file

**Comment by davidsh** [comment]
LGTM

**Comment by mellinoe** [comment]
Innerloop Windows_NT Debug Build and Test
(https://github.com/dotnet/corefx/issues/5401)

**Comment by stephentoub** [comment]
Test Innerloop Windows_NT Debug Build and Test please

**Pull request merged**